xml/OrganisationRegistrationAgency.xml

```diff
@@ -17,15 +17,17 @@
         is used to indicate Multilateral/International Agency, and XI which is
         used to indicate International Organisations. This follows ISO's
         convention of reserving codes starting with X for "private use".
+        
+         As of 17th of July 2017 the Organisation Registration Agency codelist
+         is maintained centrally by the org-id.guide project. org-id.guide provides
+         a register organization identifier lists that delivers the easiest, quickest
+         and most accurate way for open data publishers to locate organisation 
+         identifiers. IATI replicates the list directly from org-id.guide with 
+         the exception of the XI-IATI codelist which is maintained by IATI. If 
```

Comment:
I think this is confusing two concepts. 

XI-IATI just happens to be an organisation identifier list you maintain. 

It you do mention it, it would be better to mention at the end such as:

"IATI still maintains the XI-IATI organization identifier list, which is used when there are no other available lists that might be used to provide a stable organization identifier."